virtualization/windowscontainers/quick-start/quick-start-windows-server.md

```diff
@@ -18,19 +18,39 @@ This quick start is specific to Windows Server containers on Windows Server 2016
 
 **Prerequisites:**
 
-One computer system (physical or virtual) running Windows Server 2016. If you are using Windows Server 2016 TP5, please update to [Window Server 2016 Evaluation](https://www.microsoft.com/en-us/evalcenter/evaluate-windows-server-2016 ).
+One computer system (physical or virtual) running Windows Server 2016. You can get Windows Server 2016 evaluate license here [Window Server 2016 Evaluation](https://www.microsoft.com/en-us/evalcenter/evaluate-windows-server-2016 ).
 
 > Critical updates are needed in order for the Windows Container feature to function. Please install all updates before working through this tutorial.
 
-If you would like to deploy on Azure, this [template](https://github.com/Microsoft/Virtualization-Documentation/tree/master/windows-server-container-tools/containers-azure-template) makes it easy.<br/>
-<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FMicrosoft%2FVirtualization-Documentation%2Flive%2Fwindows-server-container-tools%2Fcontainers-azure-template%2Fazuredeploy.json" target="_blank">
-    <img src="http://azuredeploy.net/deploybutton.png"/>
-</a>
+Ensure your Windows Server system is up-to-date by running on elevated Powershell prompt:
+
+```
+sconfig
+```
+
+This shows a text-based configuration menu, where you can choose option 6 to Download and Install Updates:
+
+```
+===============================================================================
+                         Server Configuration
+===============================================================================
 
+1) Domain/Workgroup:                    Workgroup:  WORKGROUP
+2) Computer Name:                       WIN-HEFDK4V68M5
+3) Add Local Administrator
+4) Configure Remote Management          Enabled
 
-## 1. Install Docker
+5) Windows Update Settings:             DownloadOnly
+6) Download and Install Updates
+7) Remote Desktop:                      Disabled
+...
+```
+
+When prompted, choose option A to download all updates.
+
+## 1. Install Docker CE
```

Comment:
If these instructions are meant to cover both production and Insider then this section should be "Install Docker EE for production"